templates/automation-analytics/header-template.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+const HeaderTemplate = () => (
+  <div
+    style={{
+      fontFamily: 'Red Hat Text',
+      fontStyle: 'italic',
+      fontSize: 9,
+      color: '#ccc',
```

Comment:
We should utilize `@patternfly/react-tokens` for these values. Otherwise folks will start to complain.

Reply:
I was just migrating the template from HTML to JSX. I am gonna leave this one on respective teams.